transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/jobscheduling/AlarmManagerScheduler.java

```diff
@@ -98,7 +106,7 @@ public void schedule(TransportContext transportContext, int attemptNumber) {
 
     long backendTime = eventStore.getNextCallTime(transportContext);
 
-    long scheduleDelay =
+    long scheduleDelay = clock.getTime() +
```

Comment:
Can you add `clock.getTime()` directly to the `alarmManager.set()` so that logging below does not change?

Reply:
Done.